### DISCUSSION TITLE
Enter the world and Build terrain takes +2m when moving from overwold to neither

### DISCUSSION THREAD

**Discussion by jjziets** [itzg/docker-minecraft-bedrock-server#592] [Q&A]
On Bedrock Dedicated Server running in a docker container itzg/minecraft-bedrock-server:latest

Version: 1.21.3.01

Ubuntu 22.04.4 LTS (GNU/Linux 6.5.0-45-generic x86_64) 
Docker version 24.0.7, build 24.0.7-0ubuntu2~22.04.1

The time to enter load in to world or nether portal takes very long(+2) and cpu usage is very high.

I tried this on two machines. One is a dual xeon et-2699 v4 with 2400Mhz 128 DDR4 and raid nvme and the other is a AMD EPYC 75F3 with 3600MHz 256gb and dual pcie 4 nvme raid.

This is with no players

I set the cpu threads to 64 and even tried ram drive but it has no effect.

Is this normal? Anyway to keep nether in memory?

Nothing else on servers so ram and cpu is free

**Comment by itzg** [comment]
I'm sorry, I can't provide support for the bedrock server itself. The container image simply sets it up and runs it. Please ask on the Mojang forums about this.

**Comment by jjziets** [comment]
Ok. Just wanted to know if this is normal

**Comment by itzg** [comment]
Oh no, I meant to say "I'm sorry" originally instead of "I'm not sorry". Auto-correct/brain glitch 🤦‍♂️.

No, it doesn't seem normal so it's definitely worthwhile to look into their issues system.

**Comment by jjziets** [comment]
I think im going to try to run this bare metal and see if it has the same behaviour

**Comment by vitamins3366** [comment]
Same behavior on my machine, no problems from the windows server using same hardware, server properties, and world
